mongodbatlas/fw_data_source_mongodbatlas_project_ip_access_list_migration_test.go

```diff
@@ -0,0 +1,141 @@
+package mongodbatlas
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+)
+
+func TestAccProjectDSProjectIPAccessList_Migration_SettingIPAddress(t *testing.T) {
+	resourceName := "mongodbatlas_project_ip_access_list.test"
```

Comment:
```suggestion
	dataSourceName := "data.mongodbatlas_project_ip_access_list.test"
```
Since these migration tests are for the data source, think it might be more appropriate to verify attributes from the data source. (also applies to the following test defined in file)

Reply:
updated